[Bug] c-eval数据集下载报错

### 先决条件

- [x] 我已经搜索过 [问题](https://github.com/open-compass/opencompass/issues/) 和 [讨论](https://github.com/open-compass/opencompass/discussions) 但未得到预期的帮助。
- [x] 错误在 [最新版本](https://github.com/open-compass/opencompass) 中尚未被修复。

### 问题类型

我正在使用官方支持的任务/模型/数据集进行评估。

### 环境

'opencompass': '0.4.1+'

### 重现问题 - 代码/配置示例

from mmengine.config import read_base
with read_base():
    from opencompass.configs.datasets.ceval.ceval_gen_2daf24 import ceval_datasets  # ceval数据集

datasets = ceval_datasets

### 重现问题 - 命令或脚本

none

### 重现问题 - 错误信息

  File "XXXXX\opencompass\utils\datasets.py", line 90, in download_dataset
    assert dataset_name, f'No valid url for {data_path}!\n' + \
AssertionError: No valid url for ./data/ceval/formal_ceval!
Please make sure  `./data/ceval/formal_ceval` is correct

### 其他信息

_No response_

借楼发一个相同问题，辛苦一并帮忙看看，感谢感谢：

with read_base():
    # 直接从预设的数据集配置中读取所需的数据集配置
    from .datasets.aime2024.aime2024_gen_6e39a4 import aime2024_datasets

报错：
AssertionError: No valid url for ./data/aime.jsonl!
Please make sure  `./data/aime.jsonl` is correct

--- a/opencompass/utils/datasets_info.py
+++ b/opencompass/utils/datasets_info.py
@@ -781,6 +781,11 @@ DATASETS_URL = {
         "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/cmmlu.zip",
         "md5": "a59f4003d6918509a719ce3bc2a5d5bc",
     },
+    "/ceval/": {
+        "url":
+        "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/ceval.zip",
+        "md5": "3889202085aebd28b596b98bee43e1aa",
+    },
     "/math/": {
         "url":
         "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/math.zip",

